extensions/wikia/Email/Controller/FounderDigestController.class.php

```diff
@@ -0,0 +1,307 @@
+<?php
+
+namespace Email\Controller;
+
+use Email\EmailController;
+use Email\Check;
+use Email\Tracking\TrackingCategories;
+
+abstract class FounderDigestController extends EmailController {
+	// Defaults; will be overridden in subclasses
+	const TRACKING_CATEGORY_EN = TrackingCategories::DEFAULT_CATEGORY;
+	const TRACKING_CATEGORY_INT = TrackingCategories::DEFAULT_CATEGORY;
+
+	protected function getSummary() {
+		return $this->getMessage( 'emailext-founder-digest-summary',
+			\WikiMap::getWikiName( $this->wikiId ) )->parse();
+	}
+
+	protected function assertValidParams() {
+		$this->assertValidWikiId();
+		$this->assertPageViewsSet();
+	}
+
+	/**
+	 * Asserts that the wikiId parameter is valid
+	 *
+	 * @throws \Email\Check
+	 */
+	protected function assertValidWikiId() {
+		if ( (int) $this->wikiId < 1 ) {
```

Comment:
I don't believe you need to do this casting here. PHP does implicit casting if you do a comparison against an int like this.